### PR TITLE
This cleans up the Profile App

### LINKED ISSUE #3609: List applications user can use
Incorporate @melwoodard's comments from #3396.

### PR DESCRIPTION
**Pull request by crwallace**
This PR:
- simplifies the authorized apps list
- shows authorized apps list even when you are not verified
- This also closes #3609

Non-Verified Account:
screen shot 2016-11-07 at 1 21 43 pm

Verified Account:
screen shot 2016-11-07 at 12 03 39 pm

### CODE REVIEW

**Comment by melwoodard** [comment]
@crwallace looks great! A few suggestions:

- Add email address and change password to User Profile section instead of keeping them in a separate section
- Move change password button below email address - right now floated right, it seems like this button should be for changing the email address
- Content changes: 
  - "Member name" --> "Name"
  - Change the user's name from all uppercase to title case
  - "Authorized applications" --> "Available services"
  - Use sentence case for links to HCA and EduBen to be consistent with the other links

@AnitraAppa anything to add to the content suggestions above?

File: src/js/user-profile/components/AuthApplicationSection.jsx [hunk 1]
```diff
@@ -1,102 +1,44 @@
...
-      stuff2 = (
-        <div className="va-callout grey row">
-          <div className="medium-8 columns">
-            <p>We can not find any benefit claims information for you. Have you already submitted a claim for VA Benfits? If not please do so.</p>
-          </div>
-          <div className="medium-3 columns">
-            <button>Submit VA Claim</button>
-          </div>
+      content = (
+        <div className="info-conatiner medium-12 columns">
```
Comment (bshyong) [inline src/js/user-profile/components/AuthApplicationSection.jsx]: *container

File: src/js/user-profile/components/AuthApplicationSection.jsx [hunk 2]
```diff
@@ -1,102 +1,44 @@
...
-            <p>We can not find any prescription information for you. Have you already applied for VA Healthcare? If not please do so.</p>
-          </div>
-          <div className="medium-3 columns">
-            <button>Apply for Healthcare</button>
-          </div>
+    let content;
+
+    if (this.props.profile.accountType === 3) {
+      content = (
+        <div className="info-conatiner medium-12 columns">
```
Comment (bshyong) [inline src/js/user-profile/components/AuthApplicationSection.jsx]: *container

**Review by bshyong — APPROVED** [review]
LGTM

**Pull request merged**